src/brevitas/graph/equalize.py

```diff
@@ -159,6 +165,8 @@ def _cross_layer_equalization(srcs, sinks):
         src_broadcast_size = [1] * module.weight.ndim
         src_broadcast_size[axis] = module.weight.size(axis)
         module.weight.data = module.weight.data * torch.reshape(scaling_factors, src_broadcast_size)
+        if isinstance(module, (torch.nn.BatchNorm1d, torch.nn.BatchNorm2d, torch.nn.BatchNorm3d)):
+            module.running_mean.data = module.running_mean.data * torch.reshape(inverse_scaling_factors, src_broadcast_size)
```

Comment:
We should modify only the learned parameters to avoid interfering with further statistics collection